.github/workflows/build.yml

```diff
@@ -210,22 +210,23 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        cc:
-          - "clang"
-        cxx:
-          - "clang++"
         os:
           - "macos-13"
           - "macos-14"
           - "macos-15"
+        arch:
+          - "arm"
+          - "intel"
+        exclude:
+          - arch: "arm"
+            os: "macos-13"
+          - arch: "intel"
+            os: "macos-14"
+          - arch: "intel"
+            os: "macos-15"
```

Comment:
a bit nit-picky, but i would prefer the order swapped, eg `os` first `arch` second. reason, same order as in the matrix and the OS version the primary/functional part. the `arch` is just used for the naming of the zip.